### ISSUE TITLE
[iOS] Can't run the app locally - several issues with XCode/RN configuration

### ISSUE THREAD

**Issue by dotansimha** [MohGovIL/hamagen-react-native#37]
It seems like there is an issue while trying to run the app on for iOS (using `yarn ios`). I'm using the latest  commit from `master`.

### TL;DR

I created a PR here that fixes those issues: 

1. Run `react-native unlink react-native-background-geolocation` to fix linking issues
2. Make sure to install pods (`cd ios && pod install`)
3. Update the npm `ios` script to run: `react-native run-ios --scheme hamagen`

### How to reproduce

The first issue is that `react-native-background-geolocation` is linked manually, which is no longer needed in RN 0.60.

The log output is:

```
➜  hamagen-react-native git:(fix-deps) yarn ios 
yarn run v1.22.4
$ react-native run-ios
error React Native CLI uses autolinking for native dependencies, but the following modules are linked manually: 
  - react-native-background-geolocation (to unlink run: "react-native unlink react-native-background-geolocation")
This is likely happening when upgrading React Native from below 0.60 to 0.60 or above. Going forward, you can unlink this dependency via "react-native unlink <dependency>" and it will be included in your app automatically. If a library isn't compatible with autolinking, disregard this message and notify the library maintainers.
Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md
```

Running `react-native unlink react-native-background-geolocation` does fix the issue.

Also, documentation doesn't refer to the actual setup flow required to make iOS app work. 
For iOS, pods should be installed prior to starting rn. To fix that, it requires to do: `cd ios && pod install`

Then, there are issues with the XCode scheme - it has the incorrect name configured, and the existing schemes doesn't seems to be shared correctly, or pointing to an incorrect scheme. 

The error is:

```
xcodebuild: error: The workspace named "codeAgainstCorona" does not contain a scheme named "codeAgainstCorona". The "-list" option can be used to find the names of the schemes in the workspace.
``` 

RN looks for `codeAgainstCorona` because of mismatch in the configuration. The actual name of the scheme is `hamagen` (could be listed using `xcodebuild -workspace codeAgainstCorona.xcworkspace -list`).  

To fix that, we need to point to the correct scheme (which is missing in the repo from the first place, because it's was not shared correctly) in `ios` script in `package.json`:

```
react-native run-ios --scheme hamagen
```
